documentation/source/release_notes.rst

```diff
@@ -4,6 +4,36 @@ Release Notes
 
 All releases are available from the `GitHub Releases Page <https://github.com/cocotb/cocotb/releases>`_.
 
+cocotb 1.3
+==========
+
+Released on 1 January 2020
+
+New features
+------------
+- :func:`cocotb.triggers.with_timeout`, for a shorthand for waiting for a trigger with a timeout. (:pr:`1119`)
+- New ``timeout_time`` and ``timeout_unit`` arguments to :func:`cocotb.test`, for adding test timeouts. (:pr:`1119`)
+- The ``expect_error`` argument to :func:`cocotb.test` now accepts a specific exception type. (:pr:`1116`)
+- :envvar:`COCOTB_RESULTS_FILE`, to allow configration of the xunit XML output filename.  (:pr:`1053`)
```

Comment:
Did you intentionally leave out the ``COCOTB_HDL_TIMEUNIT`` and ``COCOTB_HDL_TIMEPRECISION`` makevars?

Reply:
No, it just wasn't labelled appropriately so I couldn't find the PR before my train arrived.

Reply:
You can use a GitHub suggestion to add this bullet here, if you like

Reply:
This is actually the first item in https://github.com/cocotb/cocotb/pull/1270#discussion_r362252238, or do you mean something else?

Reply:
I mean use the github suggestion button so that I can apply it from mobile.